Artsy/View_Controllers/Live_Auctions/ViewModels/LiveAuctionEventViewModel.swift

```diff
@@ -165,6 +165,7 @@ class LiveAuctionEventViewModel: NSObject, LiveAuctionEventViewModelType {
         case .unknown: return NSAttributedString()
         case .bidComposite: return NSAttributedString()
         case .undo: return NSAttributedString()
+        default: return NSAttributedString();
```

Comment:
_Very_ tiny nitpick:

```suggestion
        default: return NSAttributedString()
```